bake_root

```diff
@@ -2,7 +2,7 @@
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
-usage="bake -b <bookname> -r <legacy_recipes_dir> -i <inputfile> -o <outputfile>"
+usage="bake -b <bookname> -i <inputfile> -o <outputfile>"
```

Comment:
@philschatz -- `bake_root` has essentially become a wrapper for `bake`, now that it no longer delegates to easybake. could enki just call `bake` directly, and then we can remove `bake_root`?

Reply:
Yep, switching from bake_root to bake could be done in sync with this PR or as a subsequent change (marginally safer). It's just 2 lines:
https://github.com/openstax/enki/blob/main/dockerfiles/steps/git-bake.bash#L22
https://github.com/openstax/enki/blob/main/dockerfiles/steps/archive-bake.bash#L3

Reply:
discussed & will remove in a separate PR along with update to enki